src/main/java/org/folio/bulkops/service/DataImportJobCompletionReceiverService.java

```diff
@@ -38,6 +38,8 @@ public void receiveJobExecutionUpdate(DataImportJobExecution dataImportJobExecut
           return null;
         });
       }
+    } catch (Exception e) {
+      log.error("Failed to process data import event: {}.", e.getMessage());
```

Comment:
I don't think it's fully correct, module shouldn't even try to login when non-bulk-ops DI event is sent

@khandramai please suggest

Reply:
It's enough to cover unhandled issues in business logic. But, it will be nice to disable retrying also on listener configuration level adding.

Reply:
@NikitaSedyx , @siarhei-charniak , correct - we need to check if it is bulk-ops DI event. Non-bulk-ops DI events should be filtered.

Reply:
Makes sense, fixed.